### DISCUSSION TITLE
question about Prodigy-Hacking

### DISCUSSION THREAD

**Discussion by SakuraKitsunee** [Prodigy-Hacking/ProdigyMathGameHacking#926] [(A) Anything]
when did prodigy-hacking/PMGH start?

**Comment by ArcerionDev** [comment]
~september 2019

**Comment by ArcerionDev** [comment]
it didn't really grow until about February of 2020, however.

**Reply by 29wumya** [reply]
thats intresting

**Comment by SakuraKitsunee** [comment]
ohhh okay I was just interested

**Reply by ArcerionDev** [reply]
yeah that's cool, most people didn't really know about it until more recently.

**Comment by 29wumya** [comment]
I wonder why I can not battle anything in the arena.

**Comment by 29wumya** [comment]
Why can't I do the season arena when I have the prodigy hacking thing on?

**Comment by 29wumya** [comment]
Why can't I get into prodigy with the prodigy Extention on
